Helm: Optionally create a tenant (#183)
PrefectHQ/server: helm/prefect-server/README.md

See comments in `values.yaml`.

### Tenant

To automatically create a default tenant, use the flag `--set jobs.createTenant.enabled=true`. 

> A Tenant is needed for the dashboard and the agent to run properly.

Reviewer: ```suggestion
A tenant is needed for the server API to be operational.
```

Reviewer: Let's have this line first so

```
A tenant is needed for the server API to be operational.
If there is not a tenant in the database, the UI dashboard will fail to display and agents will error.

To automatically create, ...

To manually create, ...
```

### The UI loads but the dashboard is blank

If you go to the 'Home' page, it will likely direct you to create a tenant. Without a tenant, the dashboard cannot display.

> See [Tenant](#tenant) section to learn how to automatically create the default tenant during chart
> installation.

Reviewer: ```suggestion
See the [Tenant](#tenant) section to learn how to create a default tenant.
```

To create the tenant manually when your chart is already installed, 

> run `prefect backend server && prefect server create-tenant --name default --slug default` to create a default tenant.

Reviewer: Lets move this to the tenant section
